gcc/testsuite/gcc.dg/format/cmn-err-1.c

```diff
@@ -26,6 +26,8 @@ int main()
   cmn_err_func (0, "%b %s", i, "\01Foo", string);
   cmn_err_func (0, "%p", string);
   cmn_err_func (0, "%16b", i, "\01Foo");
+  cmn_err_func (0, "%hu", 0xabcd);
```

Comment:
Is there any reason to test with other modifiers?

Reply:
I just hadn't pushed the latest branch.